README.md

```diff
@@ -46,8 +46,28 @@ Sharing resources is done as follows:
      config.txt: "Hello world!"
    ```
 
-2. The resource owner creates a `SharedSecret` or `SharedConfigMap` instance to
-   make the resource shareable:
+2. The resource owner creates a `Clusterrole` to grant permission 
+to the `ServiceAccount` of `csi-driver-shared-resource` to access 
+the given resources and then creates `SharedSecret` or `SharedConfigMap` 
+instance to make the resource shareable:
+
+   ```yaml
+   apiVersion: rbac.authorization.k8s.io/v1
+   kind: ClusterRole
+   metadata:
+     name: shared-resource-secret-configmap-share-watch-sar-create
+   rules:
+     - apiGroups: [""]
+       resources: ["configmaps"]
+       resourceNames: ["shared-config"]
+       verbs: ["get", "list", "watch"]
+     - apiGroups: ["sharedresource.openshift.io"]
+       resources: ["sharedconfigmaps", "sharedsecrets"]
+       verbs: ["get", "list", "watch"]
+     - apiGroups: ["authorization.k8s.io"]
+       resources: ["subjectaccessreviews"]
+       verbs: ["create"]
+    ```
```

Comment:
We'll need an additional step to add the clusterrole to the `csi-driver-shared-resource`.
```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: openshift-builds-shared-resource-secret-configmap-share-watch-sar-create
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: openshift-builds-shared-resource-secret-configmap-share-watch-sar-create
subjects:
- kind: ServiceAccount
  name: csi-driver-shared-resource
  namespace: openshift-builds
```